flask/app/variants.py

```diff
@@ -95,62 +92,96 @@ def get_report_df(df: pd.DataFrame, type: str):
             "participant_codename",
             "family_codename",
             "dataset_id",
-            "participant_id",
-            "family_id",
         ]:
             df[col] = df[col].apply(lambda g: "; ".join(g))
         return df
 
 
-@variants_blueprint.route("/api/summary/participants", methods=["GET"])
-@login_required
-def participant_summary():
-
-    app.logger.debug("Retrieving user id..")
-
-    if app.config.get("LOGIN_DISABLED") or current_user.is_admin:
-        user_id = request.args.get("user")
-        app.logger.debug("User is admin with ID '%s'", user_id)
-    else:
-        user_id = current_user.user_id
-        app.logger.debug("User is regular with ID '%s'", user_id)
-
-    app.logger.debug("Parsing query parameters..")
-
-    genes = request.args.get("panel")
-
+def parse_gene_panel() -> List[Any]:
+    """
```

Comment:
This could maybe benefit from a more detailed function name or docstring -- it took me a minute to see that it's actually building the region query.